mathesar_ui/src/types.ts

```diff
@@ -0,0 +1 @@
+export type CssVariablesObj = Record<string, string>;
```

Comment:
I don't like this name but not able to come up with something better, open to suggestions though

Reply:
I'm fine with it. @seancolsen Do you have any suggestions?

Reply:
`CssVariablesObj` seems fine to me